Frends.Regex.IsMatch/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.2] - 2022-08-23
+### Changed
+- Documentation.
```

Comment:
It seems you have updated one of the dependencies. The changelog should reflect that change. Changes which are not visible to users or which don't affect functionality of task should not be documented.